docs/docs/recipes/gitlab-continuous-integration.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Recipes: Continuous Integration on Gitlab"
+tableOfContentsDepth: 1
+---
+
+Push your code to gitlab and automate your production build!
+
+### Prerequisites
+
+- Make sure you have the [Gatsby CLI](https://www.gatsbyjs.org/docs/gatsby-cli) installed
```

Comment:
We use relative links internally per the [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
```suggestion
- Make sure you have the [Gatsby CLI](/docs/gatsby-cli) installed
```